recipes/python-decouple/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "python-decouple" %}
+{% set version = "3.1" %}
+{% set sha256 = "1317df14b43efee4337a4aa02914bf004f010cd56d6c4bd894e6474ec8c4fe2d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - mock
+    - pytest
+    - tox
+    - docutils
+    - pygments
+
+test:
+  imports:
+    - decouple
+
+about:
+  home: https://github.com/henriquebastos/python-decouple/
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Could you send a PR upstream adding this to the distribution and put a comment with its URL here? Just need to add a `MANIFEST.in` file with the line `include LICENSE`, and possibly a `setup.cfg`, like https://github.com/torchbox/django-libsass/pull/28.

Reply:
done! thank you @dougalsutherland !